chain/chunks/src/lib.rs

```diff
@@ -606,17 +606,21 @@ impl ShardsManager {
             },
         );
 
-        if !mark_only {
-            let fetch_from_archival = chunk_needs_to_be_fetched_from_archival(
+        if mark_only {
```

Comment:
this is better reviewed with whitespace hidden

Reply:
No, that's just it, it reduces the nesting of the following code and makes some lines fit into a single line - both make it more readable.